index.html

```diff
@@ -3,47 +3,434 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>Wage Gap in Occupations</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
+    .section { padding: 1em 0; }
     .header {
       text-align: center;
+      padding: 1em 0 2em;
     }
-  </style>
+     #data-table td {
+      width: 20%;
+      /* The table has five columns, so this rule makes them of equal width. */
+    }
+    .bar {
+      height: 20px;
+      /* Note that we'll set the width dynamically using Javascript. */
+      background: #216C2A;
+    }
+    th {
+    background-color: #216C2A;
+    color: white;
+	}
+    table {border-collapse:collapse;}
+    table, th, td{
+            border-style:solid;
+            border-width:1px;
+            border-color:#000000; 
+            padding: 10px;
+         }
+   </style>
 </head>
 <body>
+<div class="header section">
+  <div class="container">    
+      <h1>Wage Gap Varies by Occupation</h1>
+      <p>Median Weekly 2015 Earnings - Current Population Survey From the <a href="http://www.bls.gov/cps/cpsaat39.htm">Bureau of Labor Statistics</a></p>
+  </div>
+</div>
+<div class="controls section">
   <div class="container">
-    <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+    <div class="btn-toolbar">
+      <div class="btn-group">
+        <button type="button" class="btn btn-default" id="sort-asc">Ascending</button>
+        <button type="button" class="btn btn-default" id="sort-desc">Descending</button>
+      </div>
+      <div class="btn-group">
+        <button type="button" class="btn btn-default" id="filter-Mgt">Management</button>
+        <button type="button" class="btn btn-default" id="filter-Sr">Sales And Related</button>
+        <button type="button" class="btn btn-default" id="filter-Drs">Healthcare Practitioners</button>
+        <button type="button" class="btn btn-default" id="filter-Admin">Office and Administrative Support</button>
```

Comment:
Purely a style consideration, but I prefer IDs and classes to be all lowercase, words separated by hyphens: `filter-mgt`. Whatever you choose to do, the important thing is to be consistent, as inconsistency leads to errors and lots of "wait, what did I put there?" It's also nice to avoid unnecessary abbreviation; for example, I'd use `filter-sales` over `filter-sr`, because there's a greater chance of me remembering what it is if I encounter it somewhere deep in the code months after writing it.